blog/2016-12-05-easy-upgrades-processed-by-git.md

```diff
@@ -59,42 +102,24 @@ All we need now is to apply this patch on your current source files. While the o
 
-React Native comes with a global CLI (the [react-native-cli](https://www.npmjs.com/package/react-native-cli) package) which delegates the command to the local CLI embedded in the `node_modules/react-native/local-cli` directory.
+This Git upgrading feature is a big improvement in developer experience it's important to make it available for any users including those who are using older versions of React Native (e.g. 0.29). By using a separate package [react-native-git-upgrade](https://www.npmjs.com/package/react-native-git-upgrade) installed globally, you can run this new code now no matter what version of React Native your app uses.
 
-As I mentioned above, the process has to be started upon your current React Native version. If we had embedded the implementation in the local-cli, you wouldn't have been able to enjoy this feature until the upcoming React Native release was in use on your project.
+One more reason is the recent [Yeoman wipeout](https://twitter.com/martinkonicek/status/800730190141857793) by Martin Konicek. We didn't want to get these Yeoman dependencies back into the `react-native` package to be able to evaluate the old template in order to create the patch.
 
-This Git upgrading feature is a huge improvement in developer experience and I really wanted it to be available for any users including those who are stuck in old versions of React Native (who said 0.28 ?) because of the painful upgrading process. That's why we came with the [react-native-git-upgrade](https://www.npmjs.com/package/react-native-git-upgrade) package, installed globally, no matter the version of React Native being used.
+## Looking ahead
 
-An other reason is the recent [Yeoman wipeout](https://twitter.com/martinkonicek/status/800730190141857793) by Martin Konicek. We didn't want to get these Yeoman dependencies back into the `react-native` package !
+The logic behind `react-native-git-upgrade` described above is going to power the standard `react-native upgrade` starting with React Native version 0.40.0.
 
-## About the future
+This means that if you are upgrading __from version 0.40.0 or higher__, you will be able to simply run `react-native upgrade` in your project without having to install anything globally. The `react-native-git-upgrade` is provided so anyone can have this improved experience now while using an older version of React Native.
 
-The standalone package is a temporary situation for a few months, it would probably never see the _1.0.0_ release: when most users will have migrated their projects to the _post-Yeoman era_ using this package, it will replace the actual `react-native upgrade` and the global package `react-native-git-upgrade` will be deprecated.
```

Comment:
Rather than talk about deprecation of this new tool when it's announced, mention how this new hotness is going to be integrated the React Native CLI.

(Positive wording)